rmw_opendds_shared_cpp/include/rmw_opendds_shared_cpp/wait.hpp

```diff
@@ -22,9 +22,41 @@
 #include "rmw/types.h"
 
 #include "rmw_opendds_shared_cpp/condition_error.hpp"
+#include "rmw_opendds_shared_cpp/event_converter.hpp"
 #include "rmw_opendds_shared_cpp/types.hpp"
 #include "rmw_opendds_shared_cpp/visibility_control.h"
 
+rmw_ret_t __handle_active_event_conditions(rmw_events_t* events)
+{
+  // enable a status condition for each event
+  if (events) {
+    for (size_t i = 0; i < events->event_count; ++i) {
+      auto current_event = static_cast<rmw_event_t*>(events->events[i]);
+      RMW_CHECK_ARGUMENT_FOR_NULL(current_event->data, RMW_RET_INVALID_ARGUMENT);
+      //DDS::Entity* dds_entity = static_cast<ConnextCustomEventInfo*>(
+      //  current_event->data)->get_entity();
+      //if (!dds_entity) {
+      //  RMW_SET_ERROR_MSG("Event handle is null");
+      //  return RMW_RET_ERROR;
+      //}
```

Comment:
What's this commented section for? Is this a TODO or dead code we don't need anymore?